runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/BenefitsSection/BenefitsSection.utils.ts

```diff
@@ -1 +1,24 @@
+import { Article } from './BenefitsArticle/BenefitsArticle.types';
+
 export const BENEFITS_TITLE_ID = 'benefits-title';
+
+export const BENEFITS_ARTICLES: Article[] = [
+    {
+        descriptionKey: 'Landing.Benefits.Slides.Repetitive.Description',
+        imageSrc: '/images/shapes/beach_access.svg',
+        imageAlt: 'beach icon',
```

Comment:
there's no need to specify in alt that it's an 'icon'. Screen reader knows and announces that.